NobelLaureateDetails/Pages/Index.cshtml.cs

```diff
@@ -14,9 +14,10 @@ public JsonResult OnGet()
         {
             List<Laureate> filteredLaureates = new List<Laureate>();
 
-            using (WebClient webClient = new WebClient())
-            {
-                string PhysicsNinetyEight = webClient.DownloadString("http://api.nobelprize.org/v1/prize.json?category=physics&year=1998");
+
+           // using (WebClient webClient = new WebClient())  //this is not required
```

Comment:
Remove this entirely.  It's considered bad practice to commit commented-out code to version control, because you can always go back in history to see what used to be there.